packages/tsconfig/test/fixtures/apps/ts-proj/Foo.ts

```
@@ -0,0 +1,13 @@
+import FooDecorator from './FooDecorator.ts';
+
+@FooDecorator()
+export class Foo {
+  demoError() {
+    try {
+    } catch (err) {
+      if (err instanceof TypeError) {
+        console.log('type error');
+      }
+    }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)